ContactList/ContactList/ViewControllerForUpdate.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  ViewControllerForUpdate.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/6/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllerForUpdate: UIViewController {
```

Comment:
When keyboard is open, it's impossible to reach email field.
Return button doesn't work